src/WebJobs.Script/Config/HostHealthMonitorOptionsSetup.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class HostHealthMonitorOptionsSetup : IConfigureOptions<HostHealthMonitorOptions>
+    {
+        private readonly IConfiguration _configuration;
+
+        public HostHealthMonitorOptionsSetup(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public void Configure(HostHealthMonitorOptions options)
+        {
+            var section = _configuration.GetSection(ConfigurationSectionNames.HealthMonitor);
```

Comment:
This is probably not grabbing the right section. It would be under `ConfigurationSectionNames.JobHost`. Similar to https://github.com/fabiocav/azure-functions-host/blob/2fd739f7540105be6ec67295d9456a2b27e535da/src/WebJobs.Script/Config/ScriptHostOptionsSetup.cs#L38-L42, `ConfigurationPath` exposes a `Combine` method to build the path with those segments. You can see how we setup the config in tests here (with the full path) https://github.com/fabiocav/azure-functions-host/blob/2fd739f7540105be6ec67295d9456a2b27e535da/test/WebJobs.Script.Tests/Configuration/ScriptHostOptionsSetupTests.cs#L204-L207

Reply:
Good catch - I thought this was working because the values in host.json are configured to the defaults, so when debugging I saw the values being populated to match host.json.